lib/modules/manager/velaci/types.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * VelaPipelineConfiguration Types
+ *
+ * Spec: https://github.com/go-vela/types/releases/latest/download/schema.json
+ * Docs: https://go-vela.github.io/docs/reference/yaml/
+ */
+export interface ObjectWithImageProp {
+  image: string;
+}
+
```

Comment:
```suggestion
export interface ObjectWithImageProp {
  image: string;
}

```

Move below `VelaPipelineConfiguration`, so the tsdoc will be shown on main interface instead of `ObjectWithImageProp` 😉